test/functional/apps/home/_sample_data.ts

```diff
@@ -19,11 +19,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const renderable = getService('renderable');
   const dashboardExpect = getService('dashboardExpect');
   const PageObjects = getPageObjects(['common', 'header', 'home', 'dashboard', 'timePicker']);
+  // Added to resolve Failing test: https://github.com/elastic/kibana/issues/89379
+  const kibanaServer = getService('kibanaServer');
 
-  // Failing: See https://github.com/elastic/kibana/issues/89379
-  describe.skip('sample data', function describeIndexTests() {
+  describe('sample data', function describeIndexTests() {
     before(async () => {
       await security.testUser.setRoles(['kibana_admin', 'kibana_sample_admin']);
+      await kibanaServer.uiSettings.update({
+        'discover:searchFieldsFromSource': true,
+      });
```

Comment:
does it fail without setting `discover:searchFieldsFromSource` to `true`?
this should not be necessary, it should return same number of records for saved searched no matter what is set here

Reply:
@kertal yes, it works!